tools/extensions/extensions_check.py

```diff
@@ -53,15 +53,15 @@
     "envoy.filters.udp_listener", "envoy.formatter", "envoy.grpc_credentials",
     "envoy.guarddog_actions", "envoy.health_checkers", "envoy.http.stateful_header_formatters",
     "envoy.internal_redirect_predicates", "envoy.io_socket", "envoy.http.original_ip_detection",
-    "envoy.matching.common_inputs", "envoy.matching.input_matchers", "envoy.tls.key_providers",
-    "envoy.quic.proof_source", "envoy.quic.server.crypto_stream", "envoy.rate_limit_descriptors",
-    "envoy.request_id", "envoy.resource_monitors", "envoy.retry_host_predicates",
-    "envoy.retry_priorities", "envoy.stats_sinks", "envoy.thrift_proxy.filters", "envoy.tracers",
-    "envoy.sip_proxy.filters", "envoy.transport_sockets.downstream",
-    "envoy.transport_sockets.upstream", "envoy.tls.cert_validator", "envoy.upstreams",
-    "envoy.wasm.runtime", "envoy.common.key_value", "envoy.network.dns_resolver",
-    "envoy.rbac.matchers", "envoy.access_loggers.extension_filters", "envoy.http.stateful_session",
-    "envoy.matching.http.input", "envoy.matching.network.input",
+    "envoy.http.header_validators", "envoy.matching.common_inputs", "envoy.matching.input_matchers",
```

Comment:
this file is now gone (hence `main` merge conflict) - the one to update is here https://github.com/envoyproxy/envoy/blob/main/tools/extensions/extensions_schema.yaml

Reply:
done